pkg/apis/operator/v1alpha1/types.go

```diff
@@ -91,6 +95,72 @@ type SettingVerticalPodAutoscaler struct {
 	Enabled *bool `json:"enabled,omitempty"`
 }
 
+// VirtualCluster contains configuration for the virtual cluster.
+type VirtualCluster struct {
+	// ETCD contains configuration for the etcds of the virtual garden cluster.
+	// +optional
+	ETCD *ETCD `json:"etcd,omitempty"`
+	// Maintenance contains information about the time window for maintenance operations.
+	Maintenance Maintenance `json:"maintenance"`
+}
+
+// ETCD contains configuration for the etcds of the virtual garden cluster.
+type ETCD struct {
+	// Main contains configuration for the main etcd.
+	// +optional
+	Main *ETCDMain `json:"main,omitempty"`
+	// Events contains configuration for the events etcd.
+	// +optional
+	Events *ETCDMain `json:"events,omitempty"`
+}
+
+// ETCDMain contains configuration for the main etcd.
+type ETCDMain struct {
+	// Backup contains the object store configuration for backups for the virtual garden etcd.
+	// +optional
+	Backup *Backup `json:"backup,omitempty"`
+	// Storage contains storage configuration.
+	// +optional
+	Storage *Storage `json:"storage,omitempty"`
+}
+
+// ETCDEvents contains configuration for the events etcd.
+type ETCDEvents struct {
```

Comment:
😵 There is no use-case for backing up `etcd-events` today. Also, I'm quite unsure whether the `etcd` component implementation would support this. I suggest to keep it like this to not get into this business now w/o a realistic scenario justifying the effort of checking this out. The current API still allows us to change it in the future if needed. OK?

Reply:
Hm, maybe not a use-case for us but others? I was actually not expecting that this is somehow different, since in the end it's just an etcd with a backup schedule and thus I preferred a unified handling in the code, but I can also understand your doubts because we never really tested this?! 😮 
If you argue that this change is too risky, then I don't want to block it at this point 👍

Reply:
I have no objections about the API change in general, but as said, I'm unsure whether the implementation supports this. It would need to checked/tried out to ensure this really works in the end.
Given that we don't do this for shoots, it's probably fair to start like this also for the `Garden`. We don't need to support this scenario with the very first version and it can still be contributed later. Hope this is fine.

Reply:
Sure, fine for me 🙂